### PR TITLE
Run  `cargo fmt` and enforce style using Travis CI

### PR DESCRIPTION
**Pull request by ArniDagur**

### CODE REVIEW

**Comment by Arcterus** [comment]
Is this actually checking everything?  This is a really small number of changes.  Also, this needs to be rebased.

**Comment by ArniDagur** [comment]
Yeah, it seems it didn't check all of the files last time. This should be all now.

File: src/env/env.rs [hunk 1]
```diff
@@ -70,8 +70,9 @@ fn parse_program_opt<'a>(opts: &mut Options<'a>, opt: &'a str) -> Result<(), i32
 }
 
 fn load_config_file(opts: &mut Options) -> Result<(), i32> {
-    // NOTE: config files are parsed using an INI parser b/c it's available and compatible with ".env"-style files
-    //   ... * but support for actual INI files, although working, is not intended, nor claimed
+    // NOTE: config files are parsed using an INI parser b/c it's available and
+    // compatible with ".env"-style files   ... * but support for actual INI
```
Comment (Arcterus) [inline src/env/env.rs]: Does `rustfmt` have a different line length for comments?  This is really bizarre IMO.

File: src/fmt/parasplit.rs [hunk 2]
```diff
@@ -217,8 +216,10 @@ pub struct Paragraph {
     init_end: usize,        // byte location of end of init in first line String
     pub indent_str: String, // string representing indent
     pub indent_len: usize,  // length of above
-    indent_end: usize, // byte location of end of indent (in crown and tagged mode, only applies to 2nd line and onward)
-    pub mail_header: bool, // we need to know if this is a mail header because we do word splitting differently in that case
+    indent_end: usize,      /* byte location of end of indent (in crown and tagged mode, only
```
Comment (Arcterus) [inline src/fmt/parasplit.rs]: Can you change the multi-line comments to single-line ones...

File: src/mkdir/mkdir.rs [hunk 3]
```diff
@@ -22,8 +22,8 @@ static NAME: &str = "mkdir";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
 /**
- * Handles option parsing
- */
+ ** Handles option parsing
```
Comment (Arcterus) [inline src/mkdir/mkdir.rs]: LOL

Can you please fix this as well?

File: src/env/env.rs (hunk 1 above)
Comment (ArniDagur) [reply src/env/env.rs]: The above line is exactly `80` characters, perhaps GitHub formatting is weird?

File: src/uptime/uptime.rs [hunk 4]
```diff
@@ -106,8 +106,8 @@ fn print_loadavg() {
 
 #[cfg(windows)]
 fn print_loadavg() {
-    // XXX: currently this is a noop as Windows does not seem to have anything comparable to
-    //      getloadavg()
+    // XXX: currently this is a noop as Windows does not seem to have anything
+    // comparable to      getloadavg()
```
Comment (ArniDagur) [inline src/uptime/uptime.rs]: This should be fixed

File: src/env/env.rs (hunk 1 above)
Comment (Arcterus) [reply src/env/env.rs]: What I mean is that the standard line length for code is 100 characters (or it used to be at least), but for comments it seems shorter?  Or maybe it's just some comments?  Not really sure.

Comment (ArniDagur) [reply src/env/env.rs]: Yeah, code is `100` characters but commens `80`.

File: src/mktemp/mktemp.rs [hunk 5]
```diff
@@ -56,8 +56,9 @@ pub fn uumain(args: Vec<String>) -> i32 {
         "DIR",
     );
     // deprecated option of GNU coreutils
-    //    opts.optflag("t", "", "Generate a template (using the supplied prefix and TMPDIR if set) \
-    //                           to create a filename template");
+    //    opts.optflag("t", "", "Generate a template (using the supplied prefix and
+    // TMPDIR if set) \                           to create a filename
```
Comment (ArniDagur) [inline src/mktemp/mktemp.rs]: I'll remove this

File: src/shred/shred.rs [hunk 6]
```diff
@@ -62,10 +60,12 @@ enum PassType<'a> {
...
 }
 
-// Used to generate all possible filenames of a certain length using NAMESET as an alphabet
+// Used to generate all possible filenames of a certain length using NAMESET as
+// an alphabet
 struct FilenameGenerator {
     name_len: usize,
-    nameset_indices: RefCell<Vec<usize>>, // Store the indices of the letters of our filename in NAMESET
+    nameset_indices: RefCell<Vec<usize>>, /* Store the indices of the letters of our filename in
+                                           * NAMESET */
```
Comment (ArniDagur) [inline src/shred/shred.rs]: I'll fix this

**Comment by ArniDagur** [comment]
I fixed the broken build

**Comment by rivy** [comment]
This is fixed in #1449.

**Pull request closed without merging**